Client/src/main/java/gui/BattleshipMainFrame.java

```diff
@@ -0,0 +1,66 @@
+package gui;
+
+import gui.panels.boards.BoardsFactory;
+import gui.panels.buttons.ButtonsPanelFactory;
+import gui.services.BoardPanelType;
+import org.springframework.beans.factory.annotation.Autowired;
+import services.shared.PlayerRegistrationService;
+
+import javax.annotation.PostConstruct;
+import javax.swing.*;
+import java.awt.*;
```

Comment:
What's wrong here? We need all those packages, and they appear automatically in this order.